### ISSUE TITLE
Find Missing Firmware, load ROM that requires it

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TASVideos/BizHawk#317]
```
What steps will reproduce the problem?
1. Have Firmware missing that is critical for a game to work.  The FDS BIOS is 
a good option as it is required.  Or Super Gameboy.
2. Open an FDS ROM.
3. When Asked "Would you like to open the firmware manager now and configure 
your firmwares?"  Select, Yes.
4. Setup the FDS Firmware (make sure it's Green in the Firmware Dialog)
5. Problem

What is the expected output? What do you see instead?
I would like EmuHawk to be smart and recognize that I setup the missing 
firmware and I would like it to continue to play the game if I have 
successfully setup the missing firmware file(s) that are required.

Instead the Core is disposed after I close the Firmware Dialog.

What version of the product are you using? On what operating system?
r8215

Please provide any additional information below.
I am unsure if this is an Enhancement or Defect.

```

Original issue reported on code.google.com by `hegyak` on 20 Nov 2014 at 12:03

**Comment by zeromus** [comment]
I want this fixed at some point

**Comment by adelikat** [comment]
scrimpeh did this already, if I am understanding the bug report.  The new behavior is that if it asks about firmware, you have the option to "Open Rom and Close" on the firmware dialog

**Comment by zeromus** [comment]
That's a weird solution. But I guess I see now why it's a bit of an odd scenario. Anyway, its good enough for now. That took care of all the thorny stuff I was imagining. Thanks Scrimpeh.

Fixed in 667e4273f38e67ddad23553ff9cd695b434c59ad
